prometheus/value.go

```diff
@@ -110,7 +110,7 @@ func NewConstMetric(desc *Desc, valueType ValueType, value float64, labelValues
 	}
 
 	metric := &dto.Metric{}
-	if err := populateMetric(valueType, value, MakeLabelPairs(desc, labelValues), nil, metric); err != nil {
+	if err := populateMetric(valueType, value, MakeLabelPairs(desc, labelValues), nil, metric, timestamppb.Now()); err != nil {
```

Comment:
I'm very confused about whether a ConstMetric should have a created timestamp or not because I'm not sure how this is supposed to be used 🤔 

Is it supposed to be used as some kind of stateless collector? I.e., would we have a new created timestamp every time we trigger the metric collection?

Reply:
Again I'm struggling to implement a test to verify 😵

Reply:
Ack, let's discuss it on our mtg today (:

Reply:
ConstMetric is interesting as indeed it's stateless - if we can't track long term creation time, perhaps we should ask called to provide it? Changing it every sample sounds wrong. ConstMetrics often represent external counters (if counter anyway).